### PR TITLE
feat: archives-page

### PR DESCRIPTION
**Pull request by tusharkandpal**
To check:

- `archives-context.js`
- services under archives folder.
- `Note.jsx` & `Archives.jsx`

### CODE REVIEW

File: src/services/authServices/archives/restoreArchiveService.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export const restoreArchiveService = async ({ id, ...note }) => {
+  console.log(id);
+  try {
+    const { data, status } = await axios.post(
+      `/api/archives/restore/${id}`,
```
Comment (mariyasada) [inline src/services/authServices/archives/restoreArchiveService.js]: remove console

**Review by mariyasada — APPROVED** [review]
This code looks clean. good to go

File: src/components/note/Note.jsx [hunk 2]
```diff
@@ -64,12 +69,34 @@ export function Note({ note, id }) {
             created on {createdDate}
           </small>
           <div className="note-icons">
-            <FiEdit2 className="note-icon" onClick={editNoteHandler} />
-            <BiArchiveIn className="note-icon" />
-            <AiOutlineDelete className="note-icon" onClick={() => deleteNoteHandler(id)} />
+            {pathname === "/notes" || pathname === "/labels" ? (
```
Comment (JantuDeb) [inline src/components/note/Note.jsx]: You can extract this condition as you are using it in multiple places

**Review by JantuDeb — APPROVED** [review]
This looks fine

File: src/components/note/Note.jsx (hunk 2 above)
Comment (tusharkandpal) [reply src/components/note/Note.jsx]: Alright will do

File: src/services/authServices/archives/restoreArchiveService.js (hunk 1 above)
Comment (tusharkandpal) [reply src/services/authServices/archives/restoreArchiveService.js]: Ya sure!

**Pull request merged**